src/components/pages/berthPage/BerthPage.tsx

```diff
@@ -95,9 +107,81 @@ class BerthPage extends Component<Props> {
     const filtered = berths.filter(filter);
     const filteredNot = berths.filterNot(filter);
     const validSelection = berths
-      .filter(berth => selectedBerthsIds.find(selectedId => selectedId === berth.id))
+      .filter(selectedBerth =>
+        selectedBerthsIds.find(selectedId => selectedId === selectedBerth.id)
+      )
       .every(filter);
 
+    const renderHarborCard: (
```

Comment:
`BerthPage` is becoming pretty big. Should this be a separate component?

Reply:
That's true, I extracted this and the corresponding one on the WS page.